### DISCUSSION TITLE
Missing the synapse-spark-kernel

### DISCUSSION THREAD

**Discussion by filipe-batista** [microsoft/SynapseVSCode#29] [Q&A]
Upon completing the pre-requisite and extension  installations I appear to still be missing the kernel when choosing one. As anyone encountered this problem?

 

![image](https://github.com/microsoft/SynapseVSCode/assets/166112834/87073b30-b956-4160-8f4c-e05186f4ff94)

**Comment by hdrenollet-csusa** [comment]
Yep, same problem here.  Completed all of the prerequisites, uninstalled and reinstalled the extension multiple times, and it's still not there

**Reply by hdrenollet-csusa** [reply]
I think I may have solved it after over 2 hours of fighting with this.  I had to add a Path environment variable for the conda installation, it didn't create during the installation.  This time, when I opened my notebook, it looks like all of the packages installed.
